src/sdk/sql_cluster_test.cc

```diff
@@ -411,7 +411,7 @@ TEST_F(SQLSDKQueryTest, GetTabletClient) {
         ASSERT_TRUE(request_row->Build());
         auto sql_cluster_router = std::dynamic_pointer_cast<SQLClusterRouter>(router);
         hybridse::sdk::Status sdk_status;
-        auto client = sql_cluster_router->GetTabletClient(db, sql, hybridse::vm::kRequestMode, request_row, sdk_status);
+        auto client = sql_cluster_router->GetTabletClient(db, sql, hybridse::vm::kRequestMode, request_row, &sdk_status);
```

Comment:
please resolve the `cpplint` error. 
`src/sdk/sql_cluster_test.cc:414:  Lines should be <= 120 characters long  [whitespace/line_length`